content/tutorials/publish-subscribe.textile

```diff
@@ -144,14 +166,30 @@ blang[ruby].
   "See this step in Github":https://github.com/ably/tutorials/commit/75becbf
 
 blang[swift].
+  If you never had a chance to build a basic user interface in Xcode, please refer to "Apple developer guide":https://developer.apple.com/library/content/referencelibrary/GettingStarted/DevelopiOSAppsSwift/Lesson2.html on how to build a simple UI and also learn more about "adding @@IBOutlets@ and @@IBActions@":https://developer.apple.com/library/content/referencelibrary/GettingStarted/DevelopiOSAppsSwift/Lesson3.html#//apple_ref/doc/uid/TP40015214-CH22-SW1.
+  To be able to subscribe to a channel add a button that will start subscription to "sports" channel. Go to your @ExampleViewController@ and add an @UIButton@ from @Object library@. Name the action "subscribeAction".
```

Comment:
Minor comment but please change copy to `To be able to subscribe to a channel, add a button that will listen for messages published on the "sports" channel.`